recipe/meta.yaml

```diff
@@ -9,25 +9,36 @@ package:
 source:
   - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
     sha256: {{ sha256 }}
+    patches:
+      - win64_detect_conda.diff  # [win]
```

Comment:
Do we still need this patch since we are not handling optional dependencies?

Reply:
Yes, because it handles CUDA `dll` loading. This is literally the first function called when we import cupy.
https://github.com/cupy/cupy/blob/cd2550f188be264086ff0d0420d706f6e0460f5a/cupy/__init__.py#L11

Reply:
Maybe I didn't answer your question, John. Optional deps are not handled in this function.

Reply:
Thanks that makes sense. I feel like I'm learning more about the inner workings of CuPy 😄

Reply:
Yes sorry didn't mean to be lecturing 😅 Though I also just learned (because of this Windows PR) that Python was notoriously bad at dll loading that in 3.8 it added [`os.add_dll_directory()`](https://docs.python.org/3/library/os.html#os.add_dll_directory) to address this issue, which the function here wraps. Living in a Linux world wouldn't anticipate these...